pages/api/sessions.ts

```diff
@@ -26,16 +21,21 @@ export default async function handler(
 
   // get the data from the body of the request
   try {
-    const session: StripeSession = await stripeClient.checkout.sessions.create({
+    const session = await stripeClient.checkout.sessions.create({
       payment_method_types: ['card'],
       mode: request.body.mode,
       line_items: [
         { price: request.body.priceId, quantity: request.body.quantity },
       ],
-      success_url: successUrl + '?session_id={CHECKOUT_SESSION_ID}',
+      success_url: successUrl + '?sessionId={CHECKOUT_SESSION_ID}',
       cancel_url: cancelUrl,
```

Comment:
since `successUrl` and `cancelUrl` are used only once, can they be inlined here? (comments can be moved down to above these 2 lines in the object